jest.unit.conf.json

```
@@ -6,8 +6,8 @@
   },
   "testPathIgnorePatterns": ["<rootDir>/frontend/test/.*/.*.tz.unit.spec.js"],
   "testMatch": [
-    "<rootDir>/frontend/test/**/*.unit.spec.js?(x)",
-    "<rootDir>/enterprise/frontend/test/**/*.unit.spec.js?(x)"
+    "<rootDir>/frontend/**/*.unit.spec.js?(x)",
+    "<rootDir>/enterprise/frontend/**/*.unit.spec.js?(x)"
```

```suggestion
    "<rootDir>/**/*.unit.spec.js?(x)",
```

If we're going to apply this approach, then it shouldn't even matter does the test live in oss or ee folder.

---

@nemanjaglumac Totally. Thank you.